eth/client.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 var (
-	RoundsPerTokenSharesClaim = big.NewInt(20)
+	RoundsPerTokenSharesClaim = big.NewInt(50)
```

Comment:
This has interesting consequences for the UX of things like using metamask to auth transactions for bonding/unbonding. In production it's almost 1 tx for every two months since the last bond/unbond/claim action. That seems borderline reasonable...so in the worst case if they haven't touched it for a year they may have to submit 7 tx's. But generally they may only to submit 1 or 2...which I think is at least tolerable for the metamask style UX. cc @jozanza 

On another note however, just because we can fit this many rounds into one claim in the gas limit on our testnet, doesn't mean that a big-gas transaction will go over well and get included quickly on ETH main net under congestion. You're not seeing a lot of 1 tx blocks these days.

Reply:
Right, in reality we should observe the estimate the ball park gas usage for particular number of rounds and then decide on a number based on that which might very well be lower than this number. I will look into doing some getting some gas reporting stats. I bumped it up here primarily for ease of use on the test network where it doesn't matter with the expectation that this number will probably change